tests/xharness/Harness.cs

```diff
@@ -307,12 +306,9 @@ void AutoConfigureIOS ()
 			var fsharp_test_suites = new string [] { "fsharp" };
 			var fsharp_library_projects = new string [] { "fsharplibrary" };
 			var bcl_suites = new string [] {
```

Comment:
Should we just remove this code at some point?

Reply:
Yes, next PR, I'm going to clean everything